src/api/controllers/likeDislikeController.js

```diff
@@ -0,0 +1,88 @@
+import models from '../models';
+import statusCode from '../../helpers/constants/status.codes';
+/**
+ * this is the LikeDislikeController controller
+ *
+ * @export
+ * @class LikeDislikeControllerController
+ */
+
+/**
+ * containing all like dislike comment 's model controllers
+ * @export
+ * @class LikeDislikeController
+ */
+export default class LikeDislikeController {
+  /**
+   * register a new
```

Comment:
Can you  write more description that can further explain what this function performs